docker-compose.yaml

```diff
@@ -0,0 +1,48 @@
+version: "3.3"
+services:
+  decluttarr:
+    image: ghcr.io/manimatter/decluttarr:latest
+    container_name: decluttarr
+    restart: always
+    environment:
+      - TIME_ZONE=Europe/Zurich
+      - PUID=1000
+      - PGID=1000
+      ## General
+      - LOG_LEVEL=INFO
+      #- TEST_RUN=True
+      #- SSL_VERIFICATION=False
+      ## Features
```

Comment:
I would not use the TZ variable to steer whether or not the timestamps are included in the logs. I think we should add a explicit setting for that, e.g. "`LOGS_WITH_TIMESTAMP`", defaulting to false and being an optional parameter.

Reply:
See my comment below